docker-entrypoint/dependencies/job/job.go

```diff
@@ -0,0 +1,41 @@
+package job
+
+import (
+	"fmt"
+
+	entry "github.com/stackanetes/docker-entrypoint/dependencies"
+	"github.com/stackanetes/docker-entrypoint/util/env"
+)
+
+type Job struct {
+	name string
+}
+
+func init() {
+	jobsDeps := env.SplitEnvToList(fmt.Sprintf("%sJOBS", entry.DependencyPrefix))
+	if jobsDeps != nil {
```

Comment:
It is best practice to make SplitEnvToList return an empty list rather than nil, so you don't need to make all these nil checks everywhere, and can simply do the loop directly 👍